### PR TITLE
Added missing rule to build the benchmark mentioned in README.

### PR DESCRIPTION
**Pull request by i-Zaak**

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -50,16 +50,19 @@ SO=$(DLLEXT)
 # artifacts {{{
 
 help:
-	echo "make tests$(EXE) | libSDDEKit.$(DLLEXT) | clean"
+	echo "make tests$(EXE) | bench_net_exc$(EXE) | libSDDEKit.$(DLLEXT) | clean"
 
 tests$(EXE): $(o_lib) $(o_test)
 	$(CC) $(CFLAGS) test/main.c $^ -o tests$(BUILD)$(EXE) $(LDFLAGS)
 
+bench_net_exc$(EXE): $(o_lib)
```
Comment (maedoc) [inline makefile]: I guess you're using Vim which folded the generic rule section, so you missed the [existing rule](https://github.com/maedoc/sddekit/blob/master/makefile#L83)?

**Comment by maedoc** [comment]
can you just remove the rule I commented on (since it's generalized by a preexisting rule) and I'll merge.

**Comment by i-Zaak** [comment]
You were right, I've misread the generic rule at the bottom -- I thought it builds the object files for the benchmarks...

**Comment by maedoc** [comment]
:bow:

**Pull request merged**